metadata-ingestion/src/datahub/cli/ingest_cli.py

```diff
@@ -233,6 +233,31 @@ def parse_restli_response(response):
     return rows
 
 
+@ingest.command()
+@click.argument("path", type=click.Path(exists=True))
+def metadata_file(path: str) -> None:
```

Comment:
This works even if you point the source to a directory that contains one or more metadata event files. So the docs should reflect that.

Reply:
updated